models/Test.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: max
+ * Date: 03.12.16
+ * Time: 21:26
+ */
+
+namespace app\models;
+
+use Yii;
+use yii\db\ActiveRecord;
+
+class Test extends ActiveRecord
+{
+    public static function tableName()
+    {
+        return 'students';
+    }
+
+    public static function showAll(){
+
+        $all = Test::find()->all();
+        return $all;
+    }
+
+    public static function showOne($id){
+        $one = Test::findOne(['id' => $id]);
+        return $one;
+
+    }
+
+    public static function createData(){
+        $new = new Test();
```

Comment:
Models should not contain methods for testing. Please move the methods createData and editData to controller and make it work as actions.